h2o-py/tests/testdir_algos/gbm/pyunit_4007_decision_tree_language_rules_printing.py

```diff
@@ -81,12 +81,13 @@ def decision_tree_language_rules_printing():
     print(tree.tree_decision_path)
     
     assert read_fixture("pyunit_4007_language_tree_representation_numerical_categorical_case.txt") == tree.tree_decision_path
-
-    print(" -- Language path representation - node ", tree.predictions.index(tree.predictions[393]), " (with pv = ", tree.predictions[393], "): -- ")
-    assert tree.decision_paths[tree.predictions.index(tree.predictions[393])] is not None
-    print(tree.decision_paths[tree.predictions.index(tree.predictions[393])])
     
-    assert read_fixture("pyunit_4007_language_path_representation_numerical_categorical_case.txt") == tree.decision_paths[tree.predictions.index(tree.predictions[393])]
+    # calculating only paths from root to leaf (not from root to 'not leaf')
```

Comment:
this part was originally testing a path from root to node which was not leaf which is not a path by definition. 
Now, only root to leaf paths are calculated, that's why testing different node.